.travis.yml

```diff
@@ -1,12 +1,44 @@
 language: python
-python: 2.7
+python:
+  - "2.7"
+  - "3.6"
+
+sudo:false
+
+cache:
+  pip: true
+  directories:
+    - $HOME/buildout-cache
+
+matrix:
+  include:
+  - python: "2.7"
+    env: PLONE_VERSION=5.1.x
+  - python: "2.7"
+    env: PLONE_VERSION=5.2.x
+  - python: "3.6"
+    env: PLONE_VERSION=5.2.x
+  exclude:
+    - python: "2.7"
+    - python: "3.6"
+
+
+before_install:
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
+  - pip install -r requirements.txt
+
 install:
   - sudo apt-get -qq install libfreetype6-dev liblcms1-dev
   - mkdir -p buildout-cache/eggs
   - mkdir -p buildout-cache/downloads
```

Comment:
These two lines above, now are redundant, you are creating these folders on the `before_install` step